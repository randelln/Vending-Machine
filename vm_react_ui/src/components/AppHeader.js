import {React, Text} from 'react';
import { Navbar, NavbarBrand } from 'reactstrap';
import '../Pages/Home/HomeStyles.css';

const AppHeader = () => {
  return (
    <Navbar color="dark" dark expand="md">
      <NavbarBrand href="/">
        <img className="logo"
          src="https://img.icons8.com/color/50/000000/vending-machine.png/"
         
          className="d-inline-block align-top"
          alt=""
        />
      
      </NavbarBrand>
    </Navbar>
  );
};

export default AppHeader;
