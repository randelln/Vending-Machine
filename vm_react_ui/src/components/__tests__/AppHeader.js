import React from 'react';
import { shallow } from 'enzyme';
import AppHeader from "../AppHeader";

describe('AppHeader', () => {
    const view = shallow(<AppHeader />);

    it('should render the AppHeader without issues', () => {
        expect(view).toMatchSnapshot();
    });
});
