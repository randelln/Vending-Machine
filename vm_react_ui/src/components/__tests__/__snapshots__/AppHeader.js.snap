// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should render the AppHeader without issues 1`] = `
<Navbar
  color="dark"
  dark={true}
  expand="md"
  tag="nav"
>
  <NavbarBrand
    href="/"
    tag="a"
  >
    <img
      alt=""
      className="d-inline-block align-top"
      src="https://img.icons8.com/color/50/000000/vending-machine.png/"
    />
  </NavbarBrand>
</Navbar>
`;
